integration-tests/sceptre-project/config/13/B.yaml

```diff
@@ -0,0 +1,3 @@
+template:
+  path: sceptre-test-artifacts/13/valid_template.json
```

Comment:
@lucasvieirasilva ..
My idea for this integration tests is to setup and tear down test artifacts in a static bucket because it's only possible to reference a known bucket name with this s3 template handler.   The automated setup and tear down of a bucket with a static name poses problems.  One, the [bucket name must be unique](https://aws.amazon.com/premiumsupport/knowledge-center/s3-error-bucket-already-exists/) which could cause conflicts when multiple users run this test.  Two, when a bucket is deleted it make take some time before the same bucket name can be used again to create a new bucket.   These issues make it difficult to create an automated integration test for this feature.

Reply:
Could a mock library like "moto" be used, or does that move it too far from proper integration tests?

Reply:
Update, I've found an approach that seems to work.  The new approach is to setup a bucket with a random name, add the template under test to the bucket, let the s3 template handler retrieve the template then when it's all done remove the artifact and test bucket.  Essentially the bucket name in this file (`sceptre-test-artifacts`) is just a place holder for the test bucket that gets created.  The test will replace the bucket path with a reference to the real bucket.  It will be something like `path: sceptre-test-artifacts-5nf3kr9/13/valid_template.json`

Reply:
@zaro0508 Nice pattern to run the integration tests for the artifacts bucket, to make it more unique you can include the account id from the AWS account which you are running the integration tests, and also the random characters.

you can use the `sts.get_caller_identity()['Account']` boto3 function.

https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sts.html#STS.Client.get_caller_identity

Reply:
Using the account id would require another request which to AWS which I would like to avoid.  If the goal is to make the test bucket name more unique then I think we should just increase the length of the random string that's  appended to the bucket name.  I have increased it from 7 to 16 characters.